src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+import { User } from '../../types/User';
+
+import './UserInfo.scss';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}` || ''}>
+    {user?.name}
+  </a>
```

Comment:
destructuring 🙏
+ strange condition: if we don't have user you add an empty link?!